test/micro/org/openjdk/bench/java/lang/foreign/StrLenTest.java

```diff
@@ -173,9 +178,13 @@ void reset() {
     }
 
     static class SlicingPool {
-        final MemorySegment pool = Arena.ofAuto().allocate(1024);
+        final MemorySegment pool;
```

Comment:
This seems ok; though I was suspicious but I can't measure a difference: with the default parameters for size of 5, 20, 100, arena was 10x-200x+ times bigger. Not it fits exactly.

I measured 1x, 10x, 100x (for size 1500 and 15000) and results appear consistent.

So, this is just a note; looks good